aioxmpp/muc/service.py

```diff
@@ -39,6 +39,21 @@
 from . import xso as muc_xso
 
 
+def _extract_one_pair(body):
```

Comment:
Yes, selecting the lowest element with respect to some consistent order sounds right.